Conclusion and future work (#54)
SpecialeBajs/speciale-rapport: article/Conclusion/conclusion.tex

\section{Conclusion}
In this work we changed the design of the layer combination for LightGCN and performed studies that showed the improvement of our method.
We proposed ALC and BLC, which on some datasets showed large improvements and on other datasets showed small differences.
ALC and BLC does layer combination based on how well each layer performed.
We also showed that in some cases it is better just to use one single convolution layer as the final embedding.
ALC and BLC was also used on node degrees, and this did not show improvements compared to ALC and BLC without considering the degree of nodes.
This should be something to consider for others deciding to continue working on weighted summation layer combination.\\

> A GCF ablation study was also conducted to investigate which parts of the model increase performance.

Reviewer: Det er vel ikke hvilke dele af modellen men men hvilke aggregation functions var optimale

> It was inconclusive which of the methods performed best, as it depended on the dataset.

Reviewer: ```suggestion
It was inconclusive which of the methods was the optimal one, as it was dependent on the dataset.
```

> LightGCN with the ALC and BLC extension were however able to consistently outperform GCF.\\

Reviewer: ```suggestion
LightGCN with either the ALC or BLC extension was however able to consistently outperform GCF.\\
```

> We believe that ALC and BLC can be used as inspiration for future research into optimizing layer combination, as it showcases that there is room for improvements on different datasets.

Reviewer: ```suggestion
We believe that ALC and BLC can be used as inspiration for future research into optimizing layer combination, as it showcases that there is room for improvements on different datasets.
The improvements are in some cases quite significant, which indicates that choosing which method to use for layer combination can have importance for the performance of the model.
This does not necessarily only apply for LightGCN, but might also have importance for other methods. 
```
